test/format/trace.c

```diff
@@ -65,16 +64,21 @@ trace_config(const char *config)
         }
         break;
     }
-    if (all)
-        g.trace_bulk = g.trace_cursor = g.trace_read = g.trace_timestamp = g.trace_txn = true;
+    if (FLD_ISSET(g.trace_flags, TRACE_ALL)) {
```

Comment:
We could define TRACE_ALL like this and we would be able to remove the code below
`#define TRACE_ALL (TRACE_BULK & TRACE_CURSOR & TRACE_READ & TRACE_TIMESTAMP & TRACE_TXN)`